components/provisioning/warewulf/SPECS/warewulf.spec

```diff
@@ -10,29 +10,35 @@
 
 ## Inserted by OHPC
 %include %{_sourcedir}/OHPC_macros
+%global pname warewulf
+## End OHPC
 
 %global debug_package %{nil}
 
-%global api 0
-
-# Base package name
-%global pname warewulf
+%if 0%{?suse_version}
+%global tftpdir /srv/tftpboot
+%else
+# Assume Fedora-based OS if not SUSE-based
+%global tftpdir /var/lib/tftpboot
+%endif
+%global srvdir %{_sharedstatedir}
 
-# Group for warewulfd and other WW operations
 %global wwgroup warewulf
 
 %if 0%{?fedora}
 %define _build_id_links none
 %endif
 
+## Inserted by OHPC
 # Service directories (change /var/lib/* default to match WW3 build)
 %global tftpdir /srv/tftpboot
```

Comment:
This variable is defined earlier depending on the distro. Maybe it should be removed here ?

Reply:
The earlier code comes directly from the original Warewulf4 spec file.  To make things easier to port between versions I put the OpenHPC code changes between comment blocks.  Since it seems that OpenHPC uses `/srv` across the board I just redefined it here as per earlier versions of the OpenHPC spec file,  leaving the previous code/definitions from the original Warewulf4 .spec file in place (easier to update with fewer modifications).

The bigger question is, should we be using `/srv` regardless or go back to the distro defaults, and in the case of Rocky/Alma, this would move it from `/srv` to `/var`.  I believe this would cause issues for upgrades.  Perhaps now is the time since adoption is probably low?

Reply:
I would not change it now anymore. We can think about it for 4.x.